### ISSUE TITLE
[med] Keyboard Navigation: Many Pages: The focus outline for first group participant is not visible

### ISSUE THREAD

**Issue by ogumen** [Expensify/App#5647]
## Action Performed:
Prerequisites:
- User has created personal account and is logged into same.
- There are 3 or more contacts added to the chat account/history.

1. Using Chrome, open https://staging.new.expensify.com/
2. Click on "+" icon and select "New Group" option.
2. Select 2 or more group participants. Participants are added to top of the list with green checkmark.
3. Use Tab/Shift+Tab to focus on the search field at the top, then press the Tab key once to focus on the first group participant and note any visual changes.

## Expected Result:
The first participant in the list should be focused after the field and must include a visible focus indicator/outline.

## Actual Result:
When there are 2 or more participants added to the group list in the panel, the focus outline for the first entry at the top always has its visible focus outline removed. Other contacts after the first have their outlines partially obscure, but still visible.

HTML:
`<div tabindex="0" class="css-1dbjc4n r-1awozwy r-1loqt21 r-1otgn73 r-1i6wzkk r-lrvibr" style="background-color: rgb(255, 255, 255); flex-direction: row; justify-content: space-between; padding-left: 20px; padding-right: 20px; text-decoration-line: none; transition-duration: 0s; -webkit-box-orient: horizontal; -webkit-box-direction: normal; -webkit-box-pack: justify;"><div class="css-1dbjc4n r-13awgt0 r-16y2uox r-1777fci" style="height: 64px; min-width: 0px; padding-bottom: 12px; padding-top: 12px; text-decoration-line: none;"><div class="css-1dbjc4n r-1awozwy" style="flex-direction: row; -webkit-box-orient: horizontal; -webkit-box-direction: normal;"><div class="css-1dbjc4n" style="height: 40px; margin-right: 16px; width: 40px;">...</div>`

CSS
`:focus-visible {
outline: 0;
box-shadow: 0px 0px 0px 1px #0185ff;
}`

Other occurrences:
1. On Chat page: Chat session history panel, first entry (Concierge).

## Workaround:
No workaround

## Area issue was found in:
Create new group overlay, Chat page

## Failed WCAG checkpoints
2.4.7

## User impact:
Keyboard users will not be able to tell if they are focusing on the element and will likely select incorrect contacts due to partial focus outline obstruction for the other elements.

## Suggested resolution:
Check the element container sizes, positioning, or possible CSS specificity conflicts as it appears something is completely obscuring the custom focus outline implementation.
https://www.w3.org/WAI/WCAG21/Techniques/failures/F78
https://darekkay.com/blog/accessible-focus-indicator/

## Platform:
Where is this issue occurring?

- Web

**Version Number:**
**Reproducible in staging?:**Yes
**Reproducible in production?:**
**Notes/Photos/Videos:** Any additional supporting documentation
**Issue reported by:**
![Bug5184465_Keyboard_Navigation-Chat-Styling_removes_focus_outline_from_first_chat_history_entry-concierge](https://user-images.githubusercontent.com/88733897/136036876-eea011ac-1497-41a5-92b7-0e71acefa2ae.png)

https://user-images.githubusercontent.com/88733897/136036890-e5fd4520-d74d-4870-bcc9-eb72d6010377.mp4

![Bug5184465_Keyboard_Navigation-Create_New_Group-Styling_removes_focus_outline_from_first_group_participant](https://user-images.githubusercontent.com/88733897/136036910-a8ead4d7-b748-4249-baae-fe22dd974aed.png)

[View all open jobs on GitHub](https://github.com/Expensify/App/issues?q=is%3Aopen+is%3Aissue+label%3A%22Help+Wanted%22)
